app/src/common/translations/en.json

```
@@ -38,7 +38,8 @@
       "headline": "Language",
       "en": "English",
       "de": "Deutsch",
-      "fr": "Français"
+      "fr": "Français",
+      "pt": "Portuguese"
```

We provide the language names in the native language (e.g. "Deutsch" instead of "German"). I assume this would be the corresponding value?

```suggestion
      "pt": "Português"
```